cram_3d_world/cram_bullet_reasoning/src/objects.lisp

```diff
@@ -280,6 +289,23 @@
 (defstruct collision-information
   rigid-body-name flags)
 
+(defmethod create-static-collision-information ((object object))
+  (if (not (object *current-bullet-world* (name object)))
+      (warn "Cannot find object named ~a" (name object))
+      (loop for body in (rigid-bodies object)
+            collecting (make-collision-information
+                        :rigid-body-name (name body)
+                        :flags (collision-flags body))
+            do (setf (collision-flags body) :cf-static-object))))
+
+(defmethod reset-collision-information ((object object) collision-information)
```

Comment:
There is a bug in bullet, where a statically spawned object stays static, regardless of the collision-flag you assign to the rigid-body. So if the object had the collision-flag `:cf-static` before and the flag is set to `nil`, the object still won't move.

To fix that, the object with the new collision-flag needs to be respawned. There is a function for that in my PR #120 , that should go in the `reset-collision-information` function. @gaya- after this PR is through I should insert the function from mine there.

Reply:
Resolved in c2f94f1